packages/app/src/__tests__/CorePack.test.tsx

```diff
@@ -55,6 +55,8 @@ test('visualize 1D dataset', async () => {
   expect(tabs[0]).toHaveTextContent(Vis.Matrix);
   expect(tabs[1]).toHaveTextContent(Vis.Line);
   expect(tabs[1]).toHaveAttribute('aria-selected', 'true');
+
+  expect(await screen.findByRole('figure', { name: 'oneD' })).toBeVisible();
```

Comment:
Setting `aria-label` instead of `aria-labelledby` on the figure (see further down) makes it possible for us to test the visualization titles.